experimental/skills/DialogToDialog/DialogRootBot/Controllers/SkillController.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Builder.Skills;
+
+namespace Microsoft.BotBuilderSamples.DialogRootBot.Controllers
+{
+    /// <summary>
+    /// A controller that handles Skill replies to the bot.
```

Comment:
```suggestion
    /// A controller that handles skill replies to the bot.
```

Reply:
Updated comment

---
In reply to: [356211073](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356211073) [](ancestors = 356211073)